django_project/geosight/data/utils.py

```diff
@@ -174,3 +177,48 @@ def run_zonal_analysis(
         elif aggregation == 'avg':
             aggregate = np.average(data) if data.size else None
         return aggregate
+
+
+
+def create_thumbnail(
```

Comment:
Please create the script that generate thumbnails for all basemap and projects, and add in new line in initialize.py
https://github.com/unicef-drp/GeoSight-OS/blob/main/django_project/initialize.py#L125